view/src/extras/table.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import MaterialTable from "material-table";
+import { forwardRef } from "react";
+
+import AddBox from "@material-ui/icons/AddBox";
+import ArrowDownward from "@material-ui/icons/ArrowDownward";
+import Check from "@material-ui/icons/Check";
+import ChevronLeft from "@material-ui/icons/ChevronLeft";
+import ChevronRight from "@material-ui/icons/ChevronRight";
+import Clear from "@material-ui/icons/Clear";
+import DeleteOutline from "@material-ui/icons/DeleteOutline";
+import Edit from "@material-ui/icons/Edit";
+import FilterList from "@material-ui/icons/FilterList";
+import FirstPage from "@material-ui/icons/FirstPage";
+import LastPage from "@material-ui/icons/LastPage";
+import Remove from "@material-ui/icons/Remove";
+import SaveAlt from "@material-ui/icons/SaveAlt";
+import Search from "@material-ui/icons/Search";
+import ViewColumn from "@material-ui/icons/ViewColumn";
+
+/** 
+ * Adds icons to all of the actions in the table. This const connects
+ * icons to certain material-table actions (forwardRef is used to get
+ * actions from children of the material-table class such as Add, Check, etc)
+ * */
+const tableIcons = {
```

Comment:
Where are all of these icons actually used in your JSX?

Reply:
The icons are loaded via the the icons prop for MaterialTable (is that what you were asking?)

Reply:
Mm, sort of.  I can see that they're registered there, but what code tells the table to actually render those icons in particular cells?